### PR TITLE
update for guzzle 6

### PR DESCRIPTION
**Pull request by buldezir**
dont forget to change package name!

### CODE REVIEW

**Comment by lzakrzewski** [comment]
Hi! Your PR is interest but Test is missing! Method "dialog" looks untestable.

File: composer.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
-  "name": "lucaszz/facebook-authentication-adapter",
+  "name": "buldezir/facebook-authentication-adapter",
   "type": "library",
   "description": "Adapter for communication with Facebook GRAPH API",
   "require": {
     "php": ">=5.4",
-    "guzzlehttp/guzzle": "~5.0"
+    "guzzlehttp/guzzle": "*"
```
Comment (krymen) [inline composer.json:7]: I'm afraid a wildcard is the bad way to define a dependency. It should be at least `6.*`.

File: src/Adapter/GuzzleFacebookApi.php [hunk 2]
```diff
@@ -41,6 +43,14 @@ public function __construct(ClientInterface $client, $redirectUri, $appId, $appS
     }
 
     /** {@inheritdoc} */
+    public function dialog()
```
Comment (krymen) [inline src/Adapter/GuzzleFacebookApi.php:46]: the implementation looks like a "old-PHP" way (`header`, `exit`). Try to use OOP and leverage one of `Response` implementation.

Comment (krymen) [inline src/Adapter/GuzzleFacebookApi.php:46]: How this method relates to upgrading Guzzle?

**Pull request closed without merging**